charts/images.yaml

```diff
@@ -42,3 +42,7 @@ images:
     sourceRepository: github.com/gardener/portmap-copier
     repository: eu.gcr.io/gardener-project/gardener/portmap-copier
     tag: v0.2.0
+  - name: alpine-iptables
+    sourceRepository: github.com/gardener/alpine-iptables
+    repository: eu.gcr.io/gardener-project/gardener/alpine-iptables
+    tag: "3.16.3"
```

Comment:
Can we use `cilium-agent` instead? It seems to have iptables included and has to be loaded anyway.
`alpine-iptables` is about to become obsolete due to https://github.com/gardener/gardener/pull/7167. Hence, I am a bit hesitant to give it a life extension here.